airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/output/ExceptionClassifier.kt

```diff
@@ -15,6 +15,16 @@ interface ExceptionClassifier : Ordered {
     val orderValue: Int
 
     override fun getOrder(): Int = orderValue
+
+    companion object {
+        fun unwind(e: Throwable, test: (Throwable) -> Boolean): Throwable? {
```

Comment:
optional - naming: test should be more like stopUnwind()

Reply:
good idea!

Reply:
done